pkgs/games/tinyfugue/default.nix

```
@@ -1,6 +1,11 @@
-{ lib, stdenv, fetchurl, ncurses, zlib
-, openssl ? null
-, sslSupport ? true
+{
+  lib,
+  stdenv,
+  fetchurl,
+  ncurses,
+  zlib,
+  openssl ? null,
```

```suggestion
  openssl,
```
that package always exists